stylus/components/button.styl

```diff
@@ -521,13 +521,15 @@ $button-subtle--regular
     themedBtnSubtle(regularTheme)
 
 $button-subtle--secondary
-    themedBtnSubtle(secondaryTheme)
+    // Not using themedBtnSubtle(secondaryTheme) because
```

Comment:
Is the comment too much? I wondered, since it's kind of an exception, it seemed legit.

Reply:
I would keep that comment IMHO 👍